packages/dds/tree/src/test/shared-tree/fuzz/fuzzEditReducers.ts

```diff
@@ -79,6 +86,31 @@ export function applySynchronizationOp(state: DDSFuzzTestState<SharedTreeFactory
 		}
 	}
 }
+function generateLeafNodeSchemas(nodeTypes: string[]) {
+	const builder = new SchemaBuilderInternal({ scope: "com.fluidframework.leaf" });
+	const leafNodeSchemas = [];
+	for (const nodeType of nodeTypes) {
+		if (
+			nodeType !== "tree2fuzz.node" &&
+			nodeType !== "com.fluidframework.leaf.number" &&
+			nodeType !== "com.fluidframework.leaf.string"
+		) {
+			if (!nodeType.startsWith("com.fluidframework.leaf")) {
+				leafNodeSchemas.push(builder.leaf(nodeType, ValueSchema.Number));
+			}
+		}
+	}
+	const library = builder.intoLibrary();
+	return { leafNodeSchemas, library };
+}
+export function applySchemaOp(state: FuzzTestState, operation: FuzzSchemaChange) {
+	const tree = state.client.channel as SharedTree;
+	const nodeTypes = getAllowableNodeTypes(state);
+	nodeTypes.push(brand(operation.contents.type));
+	const { leafNodeSchemas, library } = generateLeafNodeSchemas(nodeTypes);
+	const newSchema = createTreeStoredSchema(leafNodeSchemas, library);
```

Comment:
Is the output of `createTreeStoredSchema` really a stored schema? This makes me suspect it's actually a view schema.

Code looks fine, but maybe a rename is in order.

Reply:
Renamed to `createTreeViewSchema`